Modules/Common.lua

```diff
@@ -25,6 +26,8 @@ common.xml = require("xml")
 common.base64 = require("base64")
 common.sha1 = require("sha1")
 
+profiler = LoadModule("Modules/Profiler")
```

Comment:
I would comment this out so we don't unnecessarily load things we don't need.  Man, I'd kill for a DEBUG directive in Lua for this stuff...

Reply:
Agreed.